recipes/libheif/all/conanfile.py

```diff
@@ -0,0 +1,90 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class Libheif(ConanFile):
+    name = "libheif"
+    description = "libheif is an HEIF and AVIF file format decoder and encoder."
+    topics = ("conan", "libheif", "heif", "codec", "video")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/strukturag/libheif"
+    license = ("LGPL-3.0-only", "GPL-3.0-or-later", "MIT")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False],
+               "with_x265": [True, False]}
+    default_options = {"shared": False, "fPIC": True,
+                       "with_x265": False}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = '{}-{}'.format(self.name, self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def requirements(self):
+        self.requires("libde265/1.0.8")
+        if self.options.with_x265:
+            self.requires("libx265/3.4")
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+        os.unlink(os.path.join(self._source_subfolder, "cmake", "modules", "FindLibAOM.cmake"))
```

Comment:
Does it stop working if you don't remove this file?

Reply:
On my second thought (based on previous PRs), it would not work at all because `CMAKE_DISABLE_FIND_PACKAGE_` does not have an effect for `find_library` calls. So I reworked it to direct patching. Luckily, upstream introducted cmake options to do this in the upcoming release.

Reply:
Would it be possible to add these options to the cmake script using a patch?
Because the current `tools.replace_in_file` will not work with a new release.

Reply:
Sorry for confusion... Do not take into account my previous statement. CMAKE_DISABLE_FIND_PACKAGE_ will work fine because they ARE `find_package` calls in a root `CMakeLists.txt`. So I removed patching and just set two variables as you suggested at the first time.